test/chpldoc/functions/operatorPrimaryMethods.doc.good

```diff
@@ -0,0 +1,111 @@
+.. default-domain:: chpl
+
+.. module:: operatorPrimaryMethods.doc
+
+operatorPrimaryMethods.doc
+==========================
+**Usage**
+
+.. code-block:: chapel
+
+   use operatorPrimaryMethods.doc;
+
+
+or
+
+.. code-block:: chapel
+
+   import operatorPrimaryMethods.doc;
+
+.. class:: Foo
+
+   .. attribute:: var x: int
+
+   .. method:: operator +(a: Foo, b: Foo)
+
+   .. method:: operator chpl_align(r: range(?i), count: Foo)
+
+   .. method:: operator chpl_by(r: range(?i), count: Foo)
```

Comment:
Should those be formatted as `align` and `by`, or is the `chpl_` form how the docs should have them?

Reply:
Good point, even if it gets written with the `chpl_` prefix (which I don't think it has to be), it shouldn't display like that in the docs.